dotnet/src/Connectors/Connectors.Memory.Redis/RedisVectorStoreCollectionCreateMapping.cs

```diff
@@ -63,18 +65,47 @@ public static Schema MapToSchema(IEnumerable<VectorStoreRecordProperty> properti
             }
 
             // Data property.
-            if (property is VectorStoreRecordDataProperty dataProperty && dataProperty.IsFilterable)
+            if (property is VectorStoreRecordDataProperty dataProperty && (dataProperty.IsFilterable || dataProperty.IsFullTextSearchable))
             {
                 var storageName = storagePropertyNames[dataProperty.DataModelPropertyName];
 
-                if (dataProperty.PropertyType == typeof(string))
+                if (dataProperty.IsFilterable && dataProperty.IsFullTextSearchable)
                 {
-                    schema.AddTextField(new FieldName($"$.{storageName}", storageName));
+                    throw new InvalidOperationException($"Property '{dataProperty.DataModelPropertyName}' has both {nameof(VectorStoreRecordDataProperty.IsFilterable)} and {nameof(VectorStoreRecordDataProperty.IsFullTextSearchable)} set to true, and this is not supported.");
```

Comment:
Should we add more detail i.e. "not supported by Redis ..." so it's clear the limitation isn't one we are imposing

Reply:
Yeah, makes sense.  Updated all expect for one where the requirement is the same for all, i.e. dimensions count need to be larger than 0.